app/main/helpers/service.py

```diff
@@ -6,3 +6,12 @@ def parse_document_upload_time(data):
     match = re.search("(\d{4}-\d{2}-\d{2}-\d{2}\d{2})\..{2,3}$", data)
     if match:
         return datetime.strptime(match.group(1), "%Y-%m-%d-%H%M")
+
+
+def approve_or_download_sentence_for_framework(framework):
```

Comment:
I don't think this needs to be a helper method - we try to keep page content in the template files, and the conditional `if framework['frameworkAgreementVersion']` can be used in the template to choose between the two sentences.